### PR TITLE
CRS-1737 Add rule for 544 days

### PR DESCRIPTION
**Pull request by jbowdidge-moj**

### CODE REVIEW

File: server/services/calculateReleaseDatesService.ts [hunk 1]
```diff
@@ -283,6 +283,16 @@ export default class CalculateReleaseDatesService {
...
+    if (rules.includes('ERSED_MAX_PERIOD')) {
+      const ruleSpecificAdjustment = rulesWithExtraAdjustments.ERSED_MAX_PERIOD
+      return CalculateReleaseDatesService.createAdjustmentRow(
+        releaseDate,
+        ReleaseDateType.ERSED,
+        `${longDateFormat(unadjustedDate)} ${daysArithmeticToWords(adjustedDays)} ${arithmeticToWords(
+          ruleSpecificAdjustment
+        )}`
+      )
+    }
```
Comment (srowlands-moj) [inline server/services/calculateReleaseDatesService.ts:295]: Picky - it's only the rule specific adjustment value that changes here, could be made a method to reduce the duplication

**Review by srowlands-moj — APPROVED** [review]
Minor comment, take it or leave it

File: server/services/calculateReleaseDatesService.ts (hunk 1 above)
Comment (jbowdidge-moj) [reply server/services/calculateReleaseDatesService.ts:295]: Yes - that's picky. I thought about it but went for expedience over cleanliness.

**Comment by jbowdidge-moj** [comment]
Thanks @srowlands-moj - I left it.

**Pull request merged**